terraform/services/cost-anomaly/tofu.tf

```
@@ -0,0 +1,33 @@
+terraform {
+  backend "s3" {
+    key = "cost-anomaly/terraform.tfstate"
+  }
+}
+
+variable "region" {
+  default  = "us-east-1"
+  nullable = false
+  type     = string
+}
+
+variable "secondary_region" {
+  default  = "us-west-2"
+  nullable = false
+  type     = string
+}
+
+provider "aws" {
+  region = var.region
```

Drop the variables above and just set the region in the provider.


```suggestion
  region = "us-east-1"
```